### PR TITLE
Feature/rid 003 mysql database

### PR DESCRIPTION
**Pull request by georgebullock**
I installed MySQL, created a schema, and created a connection in the server app. Dummy route and dummy query prove it works. Need to add real tests.

Also added nodemon and some middleware (i.e. helmet) for security.

Will leave this PR open for 24 hours. If you have time, please have a look. No worries if you can't. 

You can review develop for smells when you have time and create issues for anything that smells and I was address the issue.

Cheers!

### CODE REVIEW

File: src/client/webpack.config.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // webpack v4
 const path = require('path');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const MiniCssExtractPlugin = require('../mini-css-extract-plugin');
```
Comment (Sebastian-Fitzner) [inline src/client/webpack.config.js:3]: Path is wrong, should be only the name to require a node module

File: src/server/src/app.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-
+const express = require("express");
+const helmet = require('helmet');
+const logger = require("morgan");
+const path = require("path");
+const cookieParser = require("cookie-parser");
 const app = express();
-const port = 9000
-
+const port = 9000;
```
Comment (Sebastian-Fitzner) [inline src/server/src/app.js:7]: Use dotenv for port as well

File: src/client/webpack.config.js (hunk 1 above)
Comment (georgebullock) [reply src/client/webpack.config.js:3]: Fixed. The string had no '../' on my branch. I don't remember fixing it. Strange. Anyway, it's done now.

File: src/server/src/app.js (hunk 2 above)
Comment (georgebullock) [reply src/server/src/app.js:7]: Fixed.

**Comment by georgebullock** [comment]
> Decide if you want to use single or double quotes in your source files...

Absolutely. I got into the data layer stuff before configuring the app. I created a ticket for server app config. Will add Prettier.

**Pull request merged**